File: corelib/src/main/java/com/zzh/corelib/interfaces/CommentItemClickListener.java
package com.zzh.corelib.interfaces;

/**
 * creat_user: zhengzaihong
 * Email:1096877329@qq.com
 * creat_date: 2019/12/19
 * creat_time: 11:07
 * describe 单击每条评论的事件监听
 **/
public interface CommentItemClickListener<T> {

    /**
     * 每个评论的 点击事件
     * @param itemPosition 动态的的 index
     * @param commentIndex 被评论的 评论索引
     * @param data 该条评论数据
     */
    void onClick(int itemPosition,int commentIndex,T data);
}

File: module_room/src/main/java/com/room/core/interfaces/CommentItemClickListener.kt
package com.room.core.interfaces

/**
 * create_user: zhengzaihong
 * Email:1096877329@qq.com
 * create_date: 2019/12/19
 * create_time: 11:07
 * describe 单击每条评论的事件监听
 */
open interface CommentItemClickListener {
    /**
     * 每个评论的 点击事件
     * @param itemPosition 动态的的 index
     * @param commentIndex 被评论的 评论索引
     * @param data 该条评论数据
     */
    open fun onClick(itemPosition: Int, commentIndex: Int, data: Any)
}